samples/resources-gallery/web-app/build.gradle.kts

```diff
@@ -15,8 +15,12 @@ kotlin {
                 implementation(moko.resources)
                 implementation(project(":mpp-library"))
 
-                implementation(compose.html.core)
-                implementation(compose.runtime)
+                // TODO()
+                // implementation(compose.html.core)
+                // implementation(compose.runtime)
+                implementation("org.jetbrains.compose.web:web-core:1.4.0")
+                runtimeOnly("org.jetbrains.compose.runtime:runtime:1.4.0")
```

Comment:
this dependencies moved from compose plugin?

Reply:
@KuzminVik can u help here?

Reply:
@Lobynya attention